packages/mobile/src/screens/app-drawer-screen/useAppDrawerNavigation.ts

```diff
@@ -35,17 +37,6 @@ export const useNotificationNavigation = () => {
 }
 
 export const useAppDrawerNavigation = () => {
-  const { drawerHelpers } = useContext(AppDrawerContext)
-
-  const navigation = useNavigation()
-
-  const drawerNavigation = useMemo(
-    () => ({
-      ...navigation,
-      ...drawerHelpers
-    }),
-    [drawerHelpers, navigation]
-  )
-
-  return drawerNavigation
+  const { navigation: contextNavigation } = useContext(AppTabNavigationContext)
+  return useNavigation({ customNavigation: contextNavigation })
```

Comment:
Does this change need to be behind the feature flag?

Reply:
yeah i think youre right